src/Introduction.scss

```diff
@@ -32,17 +33,32 @@ $spacer : 1rem;
     background: linear-gradient(120deg,rgba(0,0,0,50%),hsl(46, 65%, 52%))
 }
 .intro-article {
-    height: 16rem;
+    height: auto;
     border-radius: 12px;
     padding: 1rem;
 }
-.pic {
+.img-container {
+    position: relative;
+    width: 20rem;
     height: 30rem;
+    overflow: hidden;
+    margin-right: 3rem;
+}
+.responsive-image {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    object-position: center;
+    max-width: 100%;
```

Comment:
why this needed in addition to fixed width & height?

Reply:
I've encounter style issues and this seems to work at the moment. It's in progress and i haven't touch media query yet for mobile version so it will probably chage.

Reply:
can you attach screenshots with & without max-width present?

Reply:
No difference. I erased it